actionview/CHANGELOG.md

```diff
@@ -11,7 +11,9 @@
 
     *Sean Doyle*
 
-*   Rename `ActionView::TestCase::Behavior::{Content,RenderedViewContent}`
+*   Rename `ActionView::TestCase::Behavior::Content` to `ActionView::TestCase::Behavior::RenderedViewContent`
+
+    Make `RenderedViewContent` inherit from `String`. Make private API with `:nodoc:`
```

Comment:
Hmm, this line is a bit unclear to me. What is made private API?
Maybe something like:
```suggestion
    `RenderedViewContent` is explicitly made private API and inherits from `String` instead of 
    SimpleDelegator.
```